### DISCUSSION TITLE
Unable to find image 'jupyter:latest' locally

### DISCUSSION THREAD

**Discussion by arishsatheesan** [Xilinx/finn#1142] [General]
I am encountering this issue when I tried to run "bash ./run-docker.sh notebook" :
"Unable to find image 'jupyter:latest' locally
docker: Error response from daemon: pull access denied for jupyter, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'."

**Comment by auphelia** [comment]
Hi @arishsatheesan,
This looks like a Docker setup issue, can you open the interactive Docker container? Just running `bash run-docker.sh`
